README.md

```diff
@@ -1,19 +1,38 @@
 ![Build Status](https://github.com/SirVer/ultisnips/actions/workflows/main.yml/badge.svg)
```

Comment:
These are very opionated changes. Can you outline your thought process around each one?

Reply:
I wanted to prettify README and structure it better. My goal was to follow this template:

- badges: CI status, official chat, doc reference
- short project description and maybe some important notes (for instance where to get snippets in this case)
- main features
- integrations
- usage example
- installation (via Vim package system or custom package manager)
- defaults
- tutorials
- history notes (for those users who are really interested in them)

Reply:
Thanks for your suggestion, however I do not agree with this structure being better than the current README. I really appreciate you taking the time!